_data/wallscreens/perspectives.yaml

```diff
@@ -7,9 +7,11 @@ experiences:
     next: richard-weiland
   - key: richard-weiland
     next: labor
+  - key: ai-yamamoto-album
```

Comment:
not super important for now, but i think if you do set a `next:` here, we can get the attract mode to go in a loop, which iirc was the intended behavior